### PR TITLE
cve 2011 1120 and cve 2009 1441

### PR DESCRIPTION
**Pull request by jonobrien**
initial test PR :coffee:

### CODE REVIEW

File: cves/CVE-2009-1441.yml [hunk 1]
```diff
@@ -15,7 +15,17 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
```
Comment (smcintosh881) [inline cves/CVE-2009-1441.yml]: This is a really great description. It is interesting that this much information was available about this CVE.

File: cves/CVE-2009-1441.yml [hunk 2]
```diff
@@ -73,32 +85,40 @@ discovered:
...
     emerging themes?
-  commits:
-  - commit: 
-    note: 
-  - commit: 
-    note: 
+  commits: 
+  - commit: 4c870b4013b3cf658c40e2c7b98e0a419ae7236f
+    note: |
+      A lot of refactoring and decoupling of subsystems here.
```
Comment (smcintosh881) [inline cves/CVE-2009-1441.yml]: Might be interesting to add the names of the subsystems being decoupled. You mentioned above that Area-Misc Area-BrowserBackend was subsystem with the CVE. Is this the only one that was being decoupled or was their another subsystem involved?

File: cves/CVE-2009-3263.yml [hunk 3]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  Cross-site scripting (XSS) vulnerability in Google Chrome 2.x and 3.x before 3.0.195.21 
+  allows remote attackers to inject arbitrary web script or HTML via a (1) RSS or (2) Atom feed, 
+  related to the rendering of the application/rss+xml content type as XML "active content."
```
Comment (smcintosh881) [inline cves/CVE-2009-3263.yml]: This might be too complex for "outsiders to Chromium" to understand?

Comment (smcintosh881) [inline cves/CVE-2009-3263.yml]: This might be too complex for "outsiders to Chromium" to understand?

**Comment by smcintosh881** [comment]
Really nice job! It looks like there was a lot of available information about these CVEs and you took advantage of that!

File: cves/CVE-2009-1441.yml (hunk 2 above)
Comment (jonobrien) [inline cves/CVE-2009-1441.yml]: These were labelled in the code review, as the readme for the assignmnt said to look at those tags for subsystems.

File: cves/CVE-2009-3263.yml (hunk 3 above)
Comment (jonobrien) [inline cves/CVE-2009-3263.yml]: I did this before I realized we were supposed to do 2/3 of them.

File: cves/CVE-2009-1441.yml (hunk 1 above)
Comment (jonobrien) [inline cves/CVE-2009-1441.yml]: Some of this was said by the developers, some of this was content I learned while on co-op.

File: cves/CVE-2011-1120.yml [hunk 4]
```diff
@@ -37,7 +45,7 @@ fixes:
   :note: ''
```
Comment (xdmds) [inline cves/CVE-2011-1120.yml]: no quotes needed

File: cves/CVE-2011-1120.yml [hunk 5]
```diff
@@ -37,7 +45,7 @@ fixes:
...
 - :commit: d7f0642692c2ac6de15cd1e713e0c88dd740b58e
   :note: ''
-vccs: []
+vccs: [2fe6c79fdf2d51e309bfd4d6a3b21cdf5442dc69, 29a9eb570a57f7c8d2204e18bb11be2b7d565b74]
```
Comment (xdmds) [inline cves/CVE-2011-1120.yml]: remove brackets

**Pull request merged**